templates/Includes/Pagination.ss

```diff
@@ -27,9 +27,11 @@
                             </li>
                         <% else %>
                             <li class="disabled page-item">
-                                <span>
-                                    <span aria-hidden="true">&hellip;</span>
-                                </span>
+                                <a href="#"
+                                   class="page-link"
```

Comment:
Can you add this to the span so it's not detected as an anchor at all?

Reply:
Or add `disabled` class, which removes pointer events

Reply:
Actually you can do both - use a span and the disabled class, it looks right, disables pointer events and keyboard navigation

```html
<li class="page-item">
  <span class="page-link disabled" href="#">&hellip;</span>
</li>
```

Reply:
Wow okay I thought I tried all variants of span and it wasn't working, but it's totally fine 😖